packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -480,6 +480,7 @@ class DatePicker extends React.Component {
                 useExternalFormatMask={useExternalFormatMask}
               />
             )}
+            customInputRef="inputRefCallback"
```

Comment:
why is this needed?

Reply:
In later versions of `react-intl` the wrapper created by `injectIntl` is a function component. The `react-date-picker` code utilized the `ref` of the date input for focus manipulation, but the DateInput component being wrapped in a function component means it doesn't have a `ref`. There was already logic in the `react-date-picker` to pass the name of an alternative prop to pass the `ref` callback to, so I added an `inputRefCallback` prop to the DateInput that would pass it down to the `ref` of the input element and this prop indicates that's what the `react-date-picker` should use.